.obs/specfile/elemental.spec

```diff
@@ -78,21 +78,25 @@ rm -rf %{buildroot}/usr/libexec/.placeholder
 %service_add_pre elemental-populate-node-labels.service
 %service_add_pre shutdown-containerd.service
 %service_add_pre elemental-register.service
+%service_add_pre elemental-register.timer
 
 %post
 %service_add_post elemental-populate-node-labels.service
 %service_add_post shutdown-containerd.service
 %service_add_post elemental-register.service
+%service_add_pre elemental-register.timer
```

Comment:
should be `%service_add_post`. Copy & paste error I guess.

Reply:
Definitely too much copy/paste. Thank you.